### ISSUE TITLE
Add "Connection: keep-alive, Upgrade" test case

### ISSUE THREAD

**Issue by Matheus28** [crossbario/autobahn-testsuite#82]
Firefox has a slightly different header for websocket opening handshakes, mainly, its `Connection` header is:

```
Connection: keep-alive, Upgrade
```

I think it'd be interesting to add a test case for that.

**Comment by oberstet** [comment]
yep, would be good! currently, the HTTP part of the opening handshake isn't tested at all: "Opening Handshake (under development)"
